src/api/server.rkt

```diff
@@ -119,26 +146,99 @@
 
 (define (manager-tell msg . args)
```

Comment:
Why is `manager-tell` special-casing `start` behavior? The user doesn't call `manager-tell`!